File: backend/src/main/java/com/mysite/knitly/domain/mypage/dto/ReviewListItem.java
package com.mysite.knitly.domain.mypage.dto;

import java.time.LocalDate;
import java.util.List;

public record ReviewListItem(
        Long reviewId,
        Long productId,
        String productTitle,
        String productThumbnailUrl,
        Integer rating,
        String content,
        List<String> reviewImageUrls,// 프론트에서 접기/펼치기
        LocalDate createdDate
) {}

File: backend/src/main/kotlin/com/mysite/knitly/domain/mypage/dto/ReviewListItem.kt
package com.mysite.knitly.domain.mypage.dto

import java.time.LocalDate

data class ReviewListItem(
    val reviewId: Long,
    val productId: Long,
    val productTitle: String,
    val productThumbnailUrl: String?,
    val rating: Int?,
    val content: String,
    val reviewImageUrls: List<String>,
    val createdDate: LocalDate
)
